### PR TITLE
update gallery swagger and examples

### PR DESCRIPTION
**Pull request by olayemio**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ ] new API version 
      - [x] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Already supported in existing swagger.
  3. When you expect to publish swagger?  Already supported in existing swagger.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [x] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by Sandido** [comment]
We are not merging PRs in the Compute folder at this time.

**Review by yungezz — APPROVED** [review]

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/examples/CreateOrUpdateASimpleGalleryImageVersion.json [hunk 1]
```diff
@@ -16,24 +16,39 @@
               "regionalReplicaCount": 1,
               "encryption": {
```
Comment (Sandido) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/examples/CreateOrUpdateASimpleGalleryImageVersion.json:17]: If you want this in the 2021-03-01 API release, make the changes in this branch : azure-rest-api-specs/specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01 at compute-2021-03 · Azure/azure-rest-api-specs (github.com)

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/examples/CreateOrUpdateASimpleGalleryImageVersion.json [hunk 2]
```diff
@@ -16,24 +16,39 @@
...
                 "osDiskImage": {
-                  "diskEncryptionSetId": "/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/diskEncryptionSet/myDiskEncryptionSet"
```
Comment (Sandido) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/examples/CreateOrUpdateASimpleGalleryImageVersion.json:19]: These are edits to the 2019 Gallery version. @kangsun-ctrl  is moving Gallery to the 2020-09-30 version. I suggest syncing with him. https://github.com/Azure/azure-rest-api-specs/pull/12442

Edits to the 2019 version will not by default be included in the new stable 2020-09-30 version unless you make those changes.

**Review by Sandido — CHANGES_REQUESTED** [review]

**Comment by Sandido** [comment]
@olayemio , please address the comments. 
Also investigate the CI failure for .Net SDK.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/examples/CreateOrUpdateASimpleGalleryImageVersion.json (hunk 2 above)
Comment (kangsun-ctrl) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/examples/CreateOrUpdateASimpleGalleryImageVersion.json:19]: I merged gallery change into my branch. For crp/diskrp change, does Yemi need to communicate with crp/diskrp pr owner or he can do the work here in this pr?

**Comment by olayemio** [comment]
> @olayemio , please address the comments.
> Also investigate the CI failure for .Net SDK.

Not clear to me what is causing the CI failure for the .Net SDK. I only made changes to the examples for sig, diskrp and crp.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/examples/CreateOrUpdateASimpleGalleryImageVersion.json (hunk 2 above)
Comment (olayemio) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/examples/CreateOrUpdateASimpleGalleryImageVersion.json:19]: @Sandido CRP will merge my compute.json changes. Do you know if DiskRP is doing any releases for this cycle?

**Pull request closed without merging**